build.sh

```
@@ -14,37 +15,73 @@ MOD=$(basename $PWD)
-    --event close_write,move,create,delete $dir \
-    --include '\.cs$' |
-    while read changed; do
-        echo "Detected change in $changed"
-        build
-
-        cp -avf /rimworld/1.3/Mods/${MOD}/1.3 /rimworld/1.2/Mods/${MOD}
-        cp -avf /rimworld/1.4/Mods/${MOD}/1.4 /rimworld/1.2/Mods/${MOD}
-        cp -avf /rimworld/1.5/Mods/${MOD}/1.5 /rimworld/1.2/Mods/${MOD}
+    --exclude '/\.idea($|/)' \
+    --event modify,move,create,delete "$dir" "$MOD" |
+    while read event fullpath; do
```

Shellcheck (0.11.0)

[info] read without -r will mangle backslashes.

(SC2162)